packages/accordions/src/styled/accordion/StyledButton.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+export const COMPONENT_ID = 'accordions.button';
+
+export interface IStyledButton {
+  isCompact?: boolean;
+  isHovered?: boolean;
+}
+
+/**
+ * 1. Remove dotted outline from Firefox on focus.
+ */
+export const StyledButton = styled.button.attrs<IStyledButton>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledButton>`
+  outline: none;
+  border: none;
+  background: transparent;
+  padding: ${props =>
+    props.isCompact
+      ? `${props.theme.space.base * 1.5}px ${props.theme.space.base * 3}px`
+      : `${props.theme.space.base * 5}px`};
+  width: 100%;
+  text-align: ${props => (props.theme.rtl ? 'right' : 'left')};
+  color: ${props => props.isHovered && getColor('primaryHue', 600, props.theme)};
```

Comment:
There should be a `transition` property to finesse the color on hover.

Reply:
Added a transition for `color`. The common transition duration values used in Garden are `0.1s`, `0.15s`, and `0.25s`. Since `0.25s` felt slow, I went with `0.1s` transition on the color.

Reply:
It's helpful to note that Garden's `<Anchor>` uses a `0.25s` transition. Although the stacking of these elements may necessitate the quicker transition you selected. @steue please double-check that the fine-tuned result fits the Garden system and makes sense in this context.

Reply:
Update: connected with @steue and concluded that `0.1s` is a suitable duration for transitioning this `color` property.